tests/codex/sales/states/testcancelled.nim

```diff
@@ -24,7 +24,9 @@ asyncchecksuite "sales state 'cancelled'":
   var reprocessSlotWas = false
 
   setup:
-    let onCleanUp = proc(returnBytes = false, reprocessSlot = false) {.async.} =
+    let onCleanUp = proc(
+        returnBytes = false, reprocessSlot = false, currentCollateral = UInt256.none
+    ) {.async.} =
       returnBytesWas = returnBytes
       reprocessSlotWas = reprocessSlot
 
```

Comment:
We should probably check in the last test that the `returnCollateral` passed into `onCleanUp` was correct. Same in `testfinished` (though I don't think that was changed in the PR changeset, so I can't comment on that file).

Reply:
Done.